src/controllers/UserController.js

```diff
@@ -26,4 +26,41 @@ export default class UserController {
       return res.status(500).json({ error: error.message });
     }
   }
+
+  /**
+ * @name getUser
+ * @description Allows user to sign in user
+ * @param {object} req The request object
+ * @param {object} res The response object
+ * @returns {object} The API response
+ */
+  static async getUser(req, res) {
+    try {
+      const { email, password } = req.body;
+      const user = await UserServices.loginUser(email);
+
+      if (user && Utils.comparePassword(password, user.password)) {
+        delete user.password;
+        return res.status(200).json({
+          status: 200,
+          data: user
+        });
+      }
+      if (user && !Utils.comparePassword(password, user.password)) {
+        return res.status(400).json({
+          status: 400,
```

Comment:
I don't think 400 is an appropriate response code for invalid password, I think 403 or 401 would be better, you may want to contact the TTL to be sure which one to use.